ocrd-im6convert

```diff
@@ -1,10 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # shellcheck disable=SC2086
 
 set -eu
 set -o pipefail
 
-which ocrd >/dev/null 2>/dev/null || { echo "ocrd not in \$PATH. Panicking"; exit 1; }
+which ocrd >/dev/null 2>/dev/null || { echo >&2 "ocrd not in \$PATH. Panicking"; exit 1; }
+((BASH_VERSINFO[0]<4 || BASH_VERSINFO[1]<4)) && echo >&2 "bash $BASH_VERSION is too old. Panicking" && exit 1
```

Comment:
Wrong check. I also suggest a more user friendly error message, for example "bash $BASH_VERSION is too old. Please install bash 4.4 or newer."

Reply:
```suggestion
((BASH_VERSINFO<4 || BASH_VERSINFO==4 && BASH_VERSINFO[1]<4)) && echo >&2 "bash $BASH_VERSION is too old. Please install bash 4.4 or newer." && exit 1
```